src/lib/decorations/brace.typ

```diff
@@ -41,11 +43,13 @@
 ///   #show-parameter-block("amplitude", ("number"), [
 ///     Sets the height of the brace, from its baseline to its middle tip.], default: .5)
 ///   #show-parameter-block("pointiness", ("ratio", "angle"), [
-///     How pointy the spike should be. #0deg or `0%` for maximum pointiness, #90deg or `100%` for minimum.], default: 15deg)
+///     How pointy the spike should be. #0deg or `100%` for maximum pointiness, #90deg or `100%` for minimum.], default: 15deg)
```

Comment:
This mentions 100% for maximum and minimum, is this correct?

Reply:
> This mentions 100% for maximum and minimum, is this correct?

Oops.